vitrina/requests/models.py

```diff
@@ -260,4 +260,14 @@ class RequestObject(models.Model):
 class RequestAssignment(models.Model):
     organization = models.ForeignKey(Organization, models.DO_NOTHING, db_column='organization', blank=True, null=True)
     request = models.ForeignKey(Request, models.DO_NOTHING, db_column='request', blank=True, null=True)
```

Comment:
`request` ir `organization` turėtu būti unique together. Negali būti ta pati organizacija įtraukta du kartus prie to pačio poreikio.